server/src/main/java/org/opensearch/action/support/nodes/TransportNodesAction.java

```diff
@@ -209,6 +209,15 @@ protected void resolveRequest(NodesRequest request, ClusterState clusterState) {
         request.setConcreteNodes(Arrays.stream(nodesIds).map(clusterState.nodes()::get).toArray(DiscoveryNode[]::new));
     }
 
+    /**
+     * Return the concrete nodes from the request node ids which will be later used for routing requests to nodes.
+     **/
```

Comment:
Can we please add this in form of detailed java doc outlining input and output params?